deploy/examples/common.yaml

```diff
@@ -97,6 +97,9 @@ rules:
   - apiGroups: [""]
     resources: ["serviceaccounts/token"]
     verbs: ["create"]
+  - apiGroups: ["authentication.k8s.io"]
```

Comment:
Won't this have to also apply in csi-operator?
cc @Madhu-1

Reply:
https://github.com/ceph/ceph-csi-operator/pull/197 its done here